packages/docs/src/pages/startup/react.md

```diff
@@ -1,55 +1,9 @@
 ---
-title: Getting started
-weight: 0
+title: Using React Components
+weight: 3
 ---
 
-<h2 id="installation" class="ds-h2 ds-u-color--primary-darker">Installation</h2>
-
-The design system is available as NPM packages or via a .zip download.
-
-The design system consists of two packages which are installed separately.
-
-The [**core** package](https://www.npmjs.com/package/@cmsgov/design-system-core) includes the bulk of the design system:
-
-* Base styles
-* Utility classes
-* Sass/CSS and React components
-* Sass mixins and variables
-* Fonts and images
-
-```
-npm install --save @cmsgov/design-system-core
-```
-
-The [**layout** package](https://www.npmjs.com/package/@cmsgov/design-system-layout) includes:
-
-* Responsive flexbox grid framework
-
-```
-npm install --save @cmsgov/design-system-layout
-```
-
-#### Fonts and images
-
-Once you've downloaded the core package, copy the design system's `fonts` and `images` directories into the same directory as your site's CSS directory.
-
-By default, the design system expects a file structure like this:
-
-```
-├── Your site's public assets directory/
-    ├── css/
-    ├── images/
-    └── fonts/
-```
-
-You can manually copy these directories, or you could integrate this step into your build process. [Here's an example of how this step could be accomplished using a Gulp task](https://github.com/CMSgov/design-system/blob/master/examples/react-app/Gulpfile.js).
-
-You can change the default paths by overriding the following Sass variables:
-
-* `$font-path`
-* `$image-path`
-
-<h2 id="usage" class="ds-h2 ds-u-color--primary-darker">Usage</h2>
+<h2 id="installation" class="ds-h2 ds-u-color--primary-darker">Usage</h2>
```

Comment:
This page has the Using Sass/CSS and Examples sections included in it still as well as the Usage section from the Installation page (I just think those need to be cleared out, leaving only the React Components section here.)